apps/src/code-studio/components/AssetThumbnail.jsx

```diff
@@ -26,33 +27,38 @@ var assetIconStyle = {
   fontSize: '32px'
 };
 
-var AssetThumbnail = React.createClass({
+export const styles = {
+  wrapper: {
+    width: 60,
+    height: 60,
+    margin: '10px auto',
+    background: '#eee',
+    border: '1px solid #ccc',
+    textAlign: 'center'
+  },
+};
+
+var AssetThumbnail = Radium(React.createClass({
   propTypes: {
     name: React.PropTypes.string.isRequired,
-    type: React.PropTypes.oneOf(['image', 'audio', 'video', 'pdf', 'doc']).isRequired
+    type: React.PropTypes.oneOf(['image', 'audio', 'video', 'pdf', 'doc']).isRequired,
+    style: React.PropTypes.object,
+    iconStyle: React.PropTypes.object,
   },
 
   render: function () {
     var type = this.props.type;
     var name = this.props.name;
 
     return (
-      <td width="80">
-        <div className="assetThumbnail" style={{
-          width: '60px',
-          height: '60px',
-          margin: '10px auto',
-          background: '#eee',
-          border: '1px solid #ccc',
-          textAlign: 'center'
-        }}>
-          {type === 'image' ?
-             <img src={assetsApi.basePath(name)} style={assetThumbnailStyle} /> :
-             <i className={defaultIcons[type] || defaultIcons.unknown} style={assetIconStyle} />
-           }
-        </div>
-      </td>
+      <div className="assetThumbnail" style={[styles.wrapper, this.props.style]}>
+        {type === 'image' ?
+         <img src={assetsApi.basePath(name)} style={assetThumbnailStyle} /> :
+         <i className={defaultIcons[type] || defaultIcons.unknown}
+            style={[assetIconStyle, this.props.iconStyle]} />
```

Comment:
Nit: Styleguide for component layout says:
```jsx
<i
    className={defaultIcons[type] || defaultIcons.unknown}
    style={[assetIconStyle, this.props.iconStyle]}
/>
```

We should probably enable linting for this.  I think it's a combination of:
* [jsx-first-prop-new-line](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-first-prop-new-line.md): multiline
* [jsx-indent-props](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-indent-props.md): 4
* [jsx-max-props-per-line](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-max-props-per-line.md): { "maximum": 1 }